.github/workflows/e2e.yaml

```diff
@@ -29,6 +29,12 @@ jobs:
           version: v0.11.1
           image: kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
           config: .github/kind/config.yaml # disable KIND-net
+      - name: Setup testEnv binaries
```

Comment:
Please use the testenv action from here: https://github.com/fluxcd/source-controller/blob/main/.github/workflows/e2e.yaml#L29 and to allow contributors to run this locally: https://github.com/fluxcd/source-controller/blob/main/Makefile#L120

Reply:
Thank you @stefanprodan !!
That helped a lot.